### PR TITLE
Update supported-platforms-and-technologies.md

### PR DESCRIPTION
**Pull request by thisrohangupta**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/getting-started/supported-platforms-and-technologies.md [hunk 1]
```diff
@@ -139,6 +139,16 @@ import Smp from '/docs/self-managed-enterprise-edition/shared/smp-supported-plat
 
 <Smp />
 
+## Template Library
+
+#### Limitations
+
+* When you delete an existing template with active pipeline references, Harness deletes the references.
+* When you convert a runtime input in a template to a fixed value, the input type does not change in the linked pipeline. You must manually edit the linked pipeline YAML and provide the fixed values.
+* When you convert a fixed type input to a runtime input in your template, the input type does not change in the linked pipeline. You must click the template in the linked pipeline to refresh it and save the pipeline again, this re-initiates the reconciliation process.
```
Comment (brian-f-harness) [inline docs/getting-started/supported-platforms-and-technologies.md:148]:
```suggestion
* When you convert a fixed type input to a runtime input in your template, the input type does not change in the linked pipeline. You must click the template in the linked pipeline to refresh it and save the pipeline again. This re-initiates the reconciliation process.
```

File: docs/getting-started/supported-platforms-and-technologies.md [hunk 2]
```diff
@@ -139,6 +139,16 @@ import Smp from '/docs/self-managed-enterprise-edition/shared/smp-supported-plat
...
+* Chained Pipeline stages are not supported with Pipeline templates
```
Comment (brian-f-harness) [inline docs/getting-started/supported-platforms-and-technologies.md:149]:
```suggestion
* Chained pipeline stages are not supported with pipeline templates.
```

File: docs/getting-started/supported-platforms-and-technologies.md [hunk 3]
```diff
@@ -139,6 +139,16 @@ import Smp from '/docs/self-managed-enterprise-edition/shared/smp-supported-plat
...
+* When using multiple nested templates, users will need to manually reconcile the changes or force reconcile via the 3 dots menu.
```
Comment (brian-f-harness) [inline docs/getting-started/supported-platforms-and-technologies.md:150]:
```suggestion
* When using multiple nested templates, you must manually reconcile the changes or force reconcile via the 3 dots menu. 
```

File: docs/getting-started/supported-platforms-and-technologies.md [hunk 4]
```diff
@@ -139,6 +139,16 @@ import Smp from '/docs/self-managed-enterprise-edition/shared/smp-supported-plat
...
 
+## Template Library
+
+#### Limitations
+
+* When you delete an existing template with active pipeline references, Harness deletes the references.
+* When you convert a runtime input in a template to a fixed value, the input type does not change in the linked pipeline. You must manually edit the linked pipeline YAML and provide the fixed values.
+* When you convert a fixed type input to a runtime input in your template, the input type does not change in the linked pipeline. You must click the template in the linked pipeline to refresh it and save the pipeline again. This re-initiates the reconciliation process.
+* Chained pipeline stages are not supported with pipeline templates.
+* When using multiple nested templates, you must manually reconcile the changes or force reconcile via the 3 dots menu.
```
Comment (deepakbolangady) [inline docs/getting-started/supported-platforms-and-technologies.md:150]: "three-dots menu" instead?

**Review by deepakbolangady — APPROVED** [review]
Approved with a minor comment.

File: docs/getting-started/supported-platforms-and-technologies.md (hunk 4 above)
Comment (brian-f-harness) [inline docs/getting-started/supported-platforms-and-technologies.md:150]:
```suggestion
* When using multiple nested templates, you must manually reconcile the changes or force reconcile via the three-dots menu. 
```

**Pull request merged**